src/Resources/translations/VerifyEmailBundle.pl.xlf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="1">
+                <source>%count% year|%count% years</source>
+                <target>%count% roku|%count% lat</target>
```

Comment:
I'm not sure is it correct translation. Can you write some sentence with this translation to understand your intention?

Reply:
`This link will expire in 1 year` || `This link will expire in 3 years`

I'm not fluent in Polish, so please feel free to correct this if it's wrong. Thanks @mwadon

Reply:
Yes! In English, it would look like this:

> This link will expire in {{ resetToken.expirationMessageKey|trans(resetToken.expirationMessageData, 'ResetPasswordBundle') }}.

Where the `{{ }}` part will be the translation - for example "1 year" or "3 years"` in English.

Thanks!

Reply:
It's more complicated in Polish:
```
1 year - 1 rok
2,3,4 years - 2,3,4 lata
5,6,7... years - 5,6,7... lat
```

Reply:
It's not as simple in Polish as in English, but... I'm working on your above example.
1 rok
2 lata, 3 lata, 4 lata
5 lat, 6 lat and so on...

Reply:
But that's supported in Symfony already. The `pl` locale has more than 2 variants. So this should look like that: `%count% roku|%count% lata|%count% lat`

Reply:
In this particular context as @jrushlow wrote we have:

`This link will expire in 1 year` which in Polish is `Link wygaśnie w ciągu 1 roku`
`This link will expire in 3 years` which in Polish is `Link wygaśnie w ciągu 3 lat`

As @mwadon and @piotrdziubczynski indeed in Polish it is more difficult because of conjugation but in this particular context we need only two forms as it is in genitive.

`This link will expire in 1 year` which in Polish is `Link wygaśnie w ciągu 1 roku`
`This link will expire in 3 years` which in Polish is `Link wygaśnie w ciągu 3 lat`
`This link will expire in 5 years` which in Polish is `Link wygaśnie w ciągu 5 lat`
`This link will expire in 6 years` which in Polish is `Link wygaśnie w ciągu 6 lat`
`This link will expire in 17 years` which in Polish is `Link wygaśnie w ciągu 17 lat`

So the translation `%count% roku|%count% lat` is correct in this context.

Reply:
Thank you all for helping us out with this. I do however need to get a few more people to validate what the correct translations are in this context this before we can merge. Please don't take that the wrong way - from a maintainers point of view who does not understand the Polish dialect as I do in English, I can't say "yup this way is correct" or "nope that way is correct".. 

If anyone knows anyone who can review, please have them take a look - it is greatly appreciated. Thank you all for your time!

Reply:
No worries @jrushlow, I wouldn't think that way. I know that this is how it works. I would do the same. Thank you for your time developing all of this and maintaining it.

Reply:
@thomas2411

It really depends on sentence translation. My translation (and not only my) was: `Ten link wygaśnie za ...`, so `1 rok` is correct. In your translation (`Link wygaśnie w ciągu ...`) `1 roku` is also correct. So have to take just one convention.

Reply:
Indeed, that is a bummer. OK, so probably this would be a better option to have 3 different forms as it is more popular. But should be the whole sentence put in translations too so that there are no problems like this in the future? I would be for this option. Put the whole sentence int the translation to get proper context or do not put it at all and leave it up to the developer how to proceed. So just leave `getExpirationMessageData` method and skip `getExpirationMessageKey`.

Reply:
> @thomas2411
> 
> It really depends on sentence translation. My translation (and not only my) was: `Ten link wygaśnie za ...`, so `1 rok` is correct. In your translation (`Link wygaśnie w ciągu ...`) `1 roku` is also correct. So have to take just one convention.

To me

1. *wygaśnie za* is more *exact*: *wygaśnie w ciągu* sounds like a process of expiration (*expires during a year…*).
2. With the *wygaśnie za* phrase the grammar gender of the following time term is easier to express/maintain in Polish because of the accusative form that coincides with the nominative one: "rok", "miesiąc", "tydzień", "dzień" and so on have no suffixes (*za* rok => *w ciągu* rok**u**) nor fancy declesions (*za* tydzień => *w ciągu* ty**godnia**, *za* miesiąc => *w ciągu* miesiąc**a**); but: godzina [hour]: *za* godzin**ę** > *w ciągu* godzin**y**.

Reply:
I agree, but as wrote above I recommend to use the full sentence in the translation or do not translate it at all in the bundle and leave the whole to the developer. Both solutions would be good so that we do not have to spend time on discussions like these :)